<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>731</width>
      <height>451</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLineEdit" name="Z_cor">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ClearBtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>370</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Y_cor">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Y_coordinate">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Y coordinate</string>
      </property>
     </widget>
     <widget class="QLabel" name="Z_coordinate">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Z coordinate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OkButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>370</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLabel" name="X_coordinate">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>X coordinate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="X_cor">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="NextBtn">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>370</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>NEXT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="LoadBtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>501</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>INPUT RECEIVED AND STORED</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BackBtn">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>360</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DisplayBtn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Display</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
